geoportailv3/static/js/layerpermalinkservice.js

```diff
@@ -111,6 +111,14 @@ app.LayerPermalinkManager.prototype.applyLayerStateToMap_ =
         if (goog.isDefAndNotNull(node)) {
           layer = this.getLayerFunc_(node);
         } else {
+          var layerToRemove =
+              goog.array.find(addedLayers, function(addedLayer) {
+            if (addedLayer.get('queryable_id') === layerId) {
```

Comment:
Again, I find it very unfortunate that we rely on `queryable_id` as the main layer id. It would be good to create a GitHub issue to track that issue.